openverse_catalog/dags/maintenance/check_silenced_dags/check_silenced_dags.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Tuple
+
+from airflow.exceptions import AirflowException
+from airflow.models import Variable
+from common.github import GitHubAPI
+from common.slack import send_alert
+
+
+logger = logging.getLogger(__name__)
+
+
+def check_configuration(github_pat: str, airflow_variable: str):
+    silenced_dags = Variable.get(airflow_variable, {}, deserialize_json=True)
+    dags_to_reenable = get_dags_with_closed_issues(github_pat, silenced_dags)
+
+    if not dags_to_reenable:
+        logger.info(
+            "All DAGs configured to silence messages have work still in progress."
+            " No configuration updates needed."
+        )
+        return
```

Comment:
I think raising an `AirflowSkipException` here might be best, that way we have a clear view from the grid or calendar views which runs produced slack messages and which did not!

Reply:
Oh I love this! Making a note of that for the future as well 😄